apps/web/pages/insights/index.tsx

```diff
@@ -0,0 +1,446 @@
+import {
+  Card,
+  Metric,
+  Text,
+  Flex,
+  BadgeDelta,
+  DeltaType,
+  Color,
+  ColGrid,
+  LineChart,
+  Title,
+  Table,
+  TableRow,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableBody,
+  MultiSelectBox,
+  MultiSelectBoxItem,
```

Comment:
This multiselect here is actually better than our multiselect @emrysal @Jaibles https://www.tremor.so/

Reply:
agree. we could potentially fork tremor or somehow overwrite their styles to make it look even more than ours